vulnerabilities/models.py

```diff
@@ -203,6 +205,21 @@ class Meta:
         unique_together = ("package", "vulnerability")
         verbose_name_plural = "PackageRelatedVulnerabilities"
 
+class PackageRelatedVulnerabilityFix(models.Model):
+
+    package = models.ForeignKey(
+        Package, on_delete=models.CASCADE, related_name="patched_package"
+    )
+    vulnerability = models.ForeignKey(Vulnerability, on_delete=models.CASCADE)
+    source = models.TextField(null=True)
+    confidence = models.PositiveIntegerField(default=1, validators=[MinValueValidator(1), MaxValueValidator(MAX_CONFIDENCE)])
+
+    def __str__(self):
+        return f"{self.package.package_url} {self.vulnerability.vulnerability_id}"
+
+    class Meta:
+        unique_together = ("package", "vulnerability")
+        verbose_name_plural = "PackageRelatedVulnerabilitiyFixes"
```

Comment:
These are typically real names to use for display, not a camel case class name

Reply:
I could remove camel case with spaces... unless there's something better to go with ??

Reply:
nit: typo in  `PackageRelatedVulnerabilitiyFixes` -> `PackageRelatedVulnerabilityFixes`

Reply:
Why is it not called `PackageRelatedVulnerabilitiyFix` ?